superset/proxy/api.py

```diff
@@ -84,6 +94,10 @@ def error_obtaining_response(
         """
         This is a function that returns an http response based on a passed in application name
         and exception that was caught when trying to get a response from an application
+
+        :param token_name: String value representing which app we are linking to
+        :param raised_exception: Exception value representing the error that occured 
```

Comment:
occured -> occurred

Reply:
Created a PR to fix this mistake (https://github.com/CybercentreCanada/superset/pull/174)